src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/XmlKeyHelper.cs

```diff
@@ -98,8 +98,7 @@ internal static void WriteCryptoBinary(string name, int value, StringBuilder bui
             // .NET Framework compat
             if (value == 0)
             {
-                Span<byte> single = stackalloc byte[1];
-                single[0] = 0;
+                Span<byte> single = stackalloc byte[1] { 0 };
```

Comment:
```suggestion
                byte b = 0;
                Span<byte> single = new Span<byte>(ref b);
```

Why not the above?

Reply:
It can.  I didn't simply because the value isn't used here and this isn't a super hot path, so I was going for simplicity, but I can change it.